code/modules/surgery/organs/augments_arms.dm

```diff
@@ -191,16 +191,19 @@
 	name = "arm-mounted energy blade"
 	desc = "An illegal, and highly dangerous cybernetic implant that can project a deadly blade of concentrated enregy."
 	contents = newlist(/obj/item/weapon/melee/energy/blade)
+	origin_tech = "materials=5;combat=7;biotech=3;powerstorage=2;syndicate=5"
```

Comment:
These levels feel high for something that you plan on putting in the traitor uplink.
I'd need to look through all of the origin tech for the other traitor items to compare to be certain though.